RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate E2EHowItWorksView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- When is your first time in the app or after resetting your password, will shows up the blue banner wrote `Save your Encryption Password`
- Then you click on it and will appear a modal and you should click on `How it Works`
- This is the migrated view

## Screenshots

## Types of changes




## Checklist



## Further comments
